supervisor/addons/addon.py

```diff
@@ -227,6 +227,11 @@ async def load(self) -> None:
         )
 
         await self._check_ingress_port()
+
+        # Image expected not to be here during install and will be installed later
+        if initial_install:
+            return
+
```

Comment:
This work actually found a small bug. Nothing the user would notice really but we were attempting to install the image twice. We expected to install the image after a number of other steps had completed but because the load now checks for the image and installs if not found it was actually installing it much earlier.

This fixes it so it installs the image in the proper sequence.